Source/WebKit/WebProcess/WebPage/WebPageTesting.messages.in

```diff
@@ -31,4 +31,6 @@ messages -> WebPageTesting NotRefCounted {
 #endif
 
     ClearWheelEventTestMonitor()
+
+    FlushDeferredDidReceiveMouseEvent() -> () Synchronous
```

Comment:
Let's not make this synchronous.  See uses of adoptAndCallCompletionHandler.  WKPageFlushDeferredDidReceiveMouseEventForTesting will also need a void* and callback, and WebPageProxyTesting::flushDeferredDidReceiveMouseEvent will need a CallbackAggregator.